src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -96,9 +135,11 @@ Note that the billable users in lower tiers remain billable at that tier's cost,
 
 If you want access to tiered pricing, talk to your account representative. 
 
-## Opting in to the core user release [#core-acceptance]
+## Core user release [#core-acceptance]
+
+In January of 2022, we released a third [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type): the core user. New Relic organizations created after the core user release had access to that user type and the associated [changes to billing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing/#pricing-versions). Existing organizations had the option to enable core users by opting in via an acceptance form called the `core amendment`. For some organizations, this acceptance process is available in the UI, on the **Manage your plan** UI page. 
```

Comment:
@zuluecho9 How do I know if I'm one of the organizations that can access the acceptance process in the UI?

Reply:
it's pretty complicated, and it's best not to get into that.